src/Emulator.h

```diff
@@ -72,9 +72,10 @@ class TEmulator : public sigslot::has_slots<>
 		void ActionSnapLoad();
 		void ActionSnapSave();
 		void ActionROMLoad();
-		void ActionRawFile(bool save);
+        void ActionMegaRomLoad();
+        void ActionRawFile(bool save);
 
-		void ActionReset();
+        void ActionReset();
```

Comment:
Spaces instead of tabs

Reply:
damn editor (VSCode) I did not notice change, sorry for that